Frontend/CO.CDP.OrganisationApp/Pages/Forms/_FormElementMultiLineInput.cshtml

```diff
@@ -0,0 +1,32 @@
+@model FormElementMultiLineInputModel
+
+@{
+    var hasError = ((TagBuilder)Html.ValidationMessageFor(m => m.TextInput)).HasInnerHtml;
+}
+
+<div class="govuk-form-group">
+
+    @if (!string.IsNullOrWhiteSpace(Model.Heading))
+    {
+        <label class="govuk-label govuk-label--l" for="declarationInput">@Model.Heading</label>
```

Comment:
Is this label supposed to be associated with the textarea below? The `for` on the label needs to match an `id` on something.

Reply:
@andymantell well spotted Andy, I have removed it.